2025/README.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "ISAV 2025: In Situ AI, Analysis and Visualization"
+description: >
+  Information for the SC'25 conference ISAV workshop
+year: 2025
+current: true
+permalink: /2025/
+---
+
+<style type="text/css">
+
+#isav-sc25-banner {
+    margin: 2px 2px 2px 2px;
+    background: url("/2025/isav24_logo.png") 0 0 no-repeat;
+    height: 130px;
+    border: 0px solid white;
+    border-bottom: 0px solid beige;
+    align: center;
+}
+
+.container-lg > h1:first-of-type {
+    display: none;
+}
+
+p { text-align: justify; }
+
+</style>
+
+<div id="isav-sc25-banner"> </div>
+
+# ISAV 2025: In Situ AI, Analysis and Visualization
+
+<p style="text-align: left;" markdown="1">
+
+<br>
+
+*Proposed workshop* -- In conjunction with:
+[SC25, The International Conference for High Performance Computing, Networking, Storage, and Analysis](https://sc25.supercomputing.org)
+<br>
+
+</p>
+
+## Workshop Theme
+
+As HPC platforms and applications increase significantly in size, complexity, and heterogeneity, one major challenge is the widening gap between computation and our ability to gain insight from extreme-scale data and make timely, data-driven decisions.
+A well-known, yet challenging, approach is in situ processing – performing as much analysis as possible while computed data is still resident in memory.
+
+This is the 11th year of the In Situ Infrastructures for Enabling Extreme-scale Analysis and Visualization (ISAV) workshop. We celebrate that in situ processing has evolved from research efforts to a central component in supercomputer, cloud and edge applications.
+In situ methods are in high demand: in system-scale 3D visualization for the latest Exascale supercomputers, in cloud products providing responsive user experiences, in tightly coupled digital twins, and in computational sciences.
+Each one of these examples has a different set of requirements in response time, data throughput and complexity of data pipelines, and more exploration in the in situ space is needed to address multifaceted goals: (1) to preserve important elements of simulations, (2) to significantly reduce the data needed to preserve these elements, (3) to offer as much flexibility as possible for post-processing exploration, and (4) to accelerate the gathering of insights to be fast enough to make timely decisions based on it.
+
+ISAV is a community of in situ developers, practitioners, researchers, and users of in situ methods and infrastructure, connecting industry, government laboratories, and academia across all career levels.
```

Comment:
We reworked this section for the abstract today and could take this over later on.
(Here, we are usually a bit more verbose and make it 3 paragraphs instead of one.)